### PR TITLE
AsuraScansTR domain update

### LINKED ISSUE #18027: asurescanstr.com domain change
### Source information

Asura scans 1.4.52

### Source language

Turkish

### Source new URL

https://armoniscans.com/

### Other details

_No response_

### Acknowledgements

- [X] I have updated all installed extensions.
- [X] I have opened WebView and checked that the source URL is not updated yet.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by Kuuw**
Closes #18027

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Comment by kyoyacchi** [comment]
Nice! When will this update be released? I've been  waiting for it for days...

**Comment by alessandrojean** [comment]
There's no ETA.

**Comment by alessandrojean** [comment]
We've been receiving some PRs from first-time contributors that are being done through GitHub website and hence were not tested. This one have some aspects that indicates it was done through `git`, but just to be sure can you show some screenshots of it working with the new domain in the emulator?

**Comment by Kuuw** [comment]
Uploading Screen_recording_20231001_124407_1.mp4…

**Comment by Kuuw** [comment]
https://github.com/tachiyomiorg/tachiyomi-extensions/assets/67199397/aa691c11-1b0f-4aa2-b8b3-52b532dfea52

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mangathemesia/MangaThemesiaGenerator.kt [hunk 1]
```diff
@@ -14,7 +14,7 @@ class MangaThemesiaGenerator : ThemeSourceGenerator {
     override val baseVersionCode: Int = 26
 
     override val sources = listOf(
-        MultiLang("Asura Scans", "https://asuratoon.com", listOf("en", "tr"), className = "AsuraScansFactory", pkgName = "asurascans", overrideVersionCode = 29),
+        MultiLang("Asura Scans", "https://asuratoon.com", listOf("en", "tr"), className = "AsuraScansFactory", pkgName = "asurascans", overrideVersionCode = 30),
```
Comment (arkon) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mangathemesia/MangaThemesiaGenerator.kt:17]: I had to manually resolve some conflicts in this PR (albeit from the web UI), so you'll have to double check that it's working fine.

Comment (Kuuw) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mangathemesia/MangaThemesiaGenerator.kt:17]: I just checked and it seems to work fine in both En and Tr.

**Pull request merged**